runtime/binding-tcp/src/main/java/io/aklivity/zilla/runtime/binding/tcp/internal/stream/TcpClientFactory.java

```diff
@@ -115,6 +117,7 @@ public TcpClientFactory(
 
         this.initialMax = bufferPool.slotCapacity();
         this.windowThreshold = (bufferPool.slotCapacity() * config.windowThreshold()) / 100;
+        this.event = new TcpEventContext(context.supplyTypeId("tcp"), context);
```

Comment:
If we are already passing `context`, then this `supplyTypeId` can be handled in the constructor, and the `"tcp"` string should use the `TcpBinding.NAME` instead please.